infrastructure/ansible/roles/traffic_stats/templates/traffic_stats.cfg.j2

```diff
@@ -15,6 +15,7 @@
         "toUser": "{{ ts_toUser }}",
         "toPasswd": "{{ ts_toPasswd }}",
         "toUrl": "{{ ts_toUrl }}",
+        "toRequestTimeout": "{{ ts_toRequestTimeout }}",
```

Comment:
Do you need the quotes around `ts_toRequestTimeout` here?

Reply:
I don't believe so. It follows the same formatting as all of the previous fields. Unless I'm not seeing it correctly...